_maps/pubbystation.json

```diff
@@ -6,6 +6,7 @@
 	    "emergency": "emergency_pubby",
 	    "whiteship": "whiteship_pubby",
 	    "ferry": "ferry_fancy",
-	    "cargo": "cargo_box"
+		"cargo": "cargo_box",
```

Comment:
You messed up the JSON for pubby.

Reply:
This should have been removed since it's no longer in the pr